src/components/Select/Select.component.tsx

```diff
@@ -133,59 +129,43 @@ export const Select = (props: SelectProps) => {
     });
 
     return (
-        <>
-            <Wrapper>
-                <Input
-                    ref={inputRef}
-                    placeholder=' '
-                    type='text'
-                    onFocus={onFocusHandler}
-                    readOnly={true}
+        <Wrapper>
+            <Input
+                ref={inputRef}
+                placeholder=' '
```

Comment:
```suggestion
                placeholder=''
```
maybe? Or we can just skip it?